api/src/utils/validate-primary-keys.ts

```diff
@@ -0,0 +1,27 @@
+import { SchemaOverview } from '@directus/shared/types';
+import { ForbiddenException } from '../exceptions';
+import { PrimaryKey } from '../types';
+import validateUUID from 'uuid-validate';
+
+/**
+ * Validate primary keys based on its type
+ */
+export function validatePrimaryKeys(
```

Comment:
And now that it's a nice separate pure util function, we can add some unit tests too 😁

Reply:
Tooooo fast ❤️